helmfile/values/networkpolicies/service/harbor.yaml.gotmpl

```
@@ -0,0 +1,324 @@
+{{- define "old-style.rule.gen" }}{{ if and .ips .ports }}
+{{- $peers := list }}{{ range .ips }}{{ $peers = dict "cidr" . | append $peers }}{{ end }}
+{{- $ports := list }}{{ range .ports }}{{ $ports = dict "tcp" . | append $ports }}{{ end }}
+{{- dict "peers" $peers "ports" $ports | toYaml | nindent 4 | trim -}}
+{{- end }}{{ end -}}
+
+rules:
+
+  egress-rule-harbor-core:
+    peers:
+      - podSelectorLabels:
+          component: core
+    ports:
+      - tcp: 8080
+
+  egress-rule-harbor-database:
+    {{- if eq .Values.harbor.database.type "internal" }}
+    peers:
+      - podSelectorLabels:
+          component: database
+    ports:
+      - tcp: 5432
+    {{- else if eq .Values.harbor.database.type "external" }}
+    {{- with .Values.networkPolicies.harbor.database.externalEgress }}
+    peers:
+      {{ toYaml .peers | nindent 8 }}
+    ports:
+      {{- range .ports }}
+        - tcp: {{ . }}
+      {{- end }}
+    {{- end }}
+    {{- end }}
+
+  egress-rule-harbor-redis:
+    peers:
+    {{- if eq .Values.harbor.redis.type "internal" }}
+      - podSelectorLabels:
+          component: redis
+    ports:
+      - tcp: 6379
+    {{- else if eq .Values.harbor.redis.type "external" }}
+    {{- with .Values.networkPolicies.harbor.redis.externalEgress }}
+    peers:
+      {{ toYaml .peers | nindent 8 }}
+    ports:
+      {{- range .ports }}
+        - tcp: {{ . }}
+      {{- end }}
+    {{- end }}
+    {{- end }}
+
+  egress-rule-harbor-registry:
+    peers:
+      - podSelectorLabels:
+          component: registry
+    ports:
+      - tcp: 5000
+      - tcp: 8080
+
+  egress-rule-harbor-trivy:
+    peers:
+      - podSelectorLabels:
+          component: trivy
+    ports:
+      - tcp: 8080
+
+  egress-rule-harbor-portal:
+    peers:
+      - podSelectorLabels:
+          component: portal
+    ports:
+      - tcp: 8080
+
+  egress-rule-harbor-notary-server:
+    peers:
+      - podSelectorLabels:
+          component: notary-server
+    ports:
+      - tcp: 4443
+
+  egress-rule-harbor-jobservice:
+    peers:
+      - podSelectorLabels:
+          component: jobservice
+    ports:
+      - tcp: 8080
+
+  {{- with .Values.networkPolicies.harbor.registries }}
+  egress-rule-harbor-registries:
```

```suggestion
  egress-rule-harbor-external-registries:
```
To be more inline with the other two rules.